src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -22,6 +22,8 @@
 
   <PropertyGroup>
     <TestWithCore Condition="'$(TestWithCore)' == '' And '$(TestTFM)' != 'netcoreapp1.0'">false</TestWithCore>
+    <!-- TestAppX is true only for debug scenario which runs on non-aot runtime ids -->
```

Comment:
So do you mean the aot scenario of installing a native compiled test as an installed AppX? This needs a separate scenario to distinguish this from the just running the native compiled tests that use xunit.console.netcore.exe to run xunit. I can add this when we start testing that particular scenario, basically changing the TestAppX boolean to something else (TestUWPDebug/TestUWPRelease).